packages/react-graphql/src/hooks/query.ts

```diff
@@ -38,12 +38,16 @@ export default function useQuery<
   } = options;
   const client = useApolloClient(overrideClient);
 
-  if (typeof window === 'undefined' && skip) {
+  if ((typeof window === 'undefined' && skip) || fetchPolicy === 'no-cache') {
```

Comment:
I think I'm just not getting it, but should this be:
```
if (typeof window === 'undefined' && (skip || fetchPolicy === 'no-cache')) {
```

Reply:
Yup, you're absolutely right